src/components/Layouts/Primary/TopBar/Items/AddMenu/Items/CreateContact/CreateContact.tsx

```diff
@@ -0,0 +1,208 @@
+import React, { ReactElement } from 'react';
+import {
+  Button,
+  CircularProgress,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  FormControl,
+  FormLabel,
+  Grid,
+  IconButton,
+  styled,
+  TextField,
+} from '@material-ui/core';
+import CloseIcon from '@material-ui/icons/Close';
+import { useTranslation } from 'react-i18next';
+import { Formik } from 'formik';
+import * as yup from 'yup';
+import { useSnackbar } from 'notistack';
+import { useRouter } from 'next/router';
+import {
+  ContactCreateInput,
+  LikelyToGiveEnum,
+  PledgeFrequencyEnum,
+  PreferredContactMethodEnum,
+  SendNewsletterEnum,
+  StatusEnum,
+} from '../../../../../../../../../graphql/types.generated';
+import { useCreateContactMutation } from './CreateContact.generated';
+
+interface Props {
+  accountListId: string;
+  handleClose: () => void;
+}
+
+const CreateContactTitle = styled(DialogTitle)(() => ({
+  textTransform: 'uppercase',
+  textAlign: 'center',
+}));
+
+const CloseButton = styled(IconButton)(({ theme }) => ({
+  position: 'absolute',
+  right: theme.spacing(1),
+  top: theme.spacing(1),
+  color: theme.palette.text.primary,
+  '&:hover': {
+    backgroundColor: '#EBECEC',
+  },
+}));
+
+const LogFormControl = styled(FormControl)(() => ({
+  width: '100%',
+}));
+
+const LogFormLabel = styled(FormLabel)(({ theme }) => ({
+  margin: theme.spacing(1, 0),
+  fontWeight: 'bold',
+  color: theme.palette.primary.dark,
+  '& span': {
+    color: theme.palette.error.main,
+  },
+}));
+
+const LogTextField = styled(TextField)(({ theme }) => ({
+  '& div': {
+    padding: theme.spacing(1),
+  },
+}));
+
+const LoadingIndicator = styled(CircularProgress)(({ theme }) => ({
+  margin: theme.spacing(0, 1, 0, 0),
+}));
+
+const contactSchema: yup.SchemaOf<ContactCreateInput> = yup.object({
```

Comment:
This may be overkill when the only field needed is ```name```.... but I based it on how other forms are handled.

Reply:
Is this ever going to be used/reused? Seems like having extra fields means extra maintenance for the future.